OpenTrace/Info.plist

```diff
@@ -96,6 +96,7 @@
 	</array>
 	<key>UIBackgroundModes</key>
 	<array>
+		<string>audio</string>
```

Comment:
not sure why this change was made. think it was something to do with the provisioning profile.